Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -98,7 +98,7 @@ public unsafe InputUpdateDelegate onUpdate
                             eventBufferPtr->sizeInBytes = 0;
                         }
                     };
-                    #elif UNITY_2019_1
+                    #elif UNITY_2018_3_OR_NEWER
```

Comment:
We can simply nuke the other codepath. 2018.3 now being our min version, we don't need explicit UNITY_2018_3_OR_NEWER blocks anymore.

Reply:
@Rene-Damm fixed.